### DISCUSSION TITLE
Provide open/close buttons for left/right drawers

### DISCUSSION THREAD

**Discussion by v479038280** [zauberzeug/nicegui#1253] [Ideas / Feature Requests]
When we use the side navigation bar, can we add a button by default, which can be used to  close  and open the side menu
like this below

![微信截图_20230725092229](https://github.com/zauberzeug/nicegui/assets/25096045/ab441b5b-26bb-412a-a0de-bbd69a78dff7)

![微信截图_20230725092243](https://github.com/zauberzeug/nicegui/assets/25096045/1aa586f7-563b-44ea-8690-8b331d70c83e)

**Comment by rodja** [comment]
Interesting suggestion. If implemented, it should be possible to disable it with a simple flag. There are other ways to open/close a side bar. We need to walk the line of "gentle learning curve and great to customize".

**Reply by v479038280** [reply]
Yes ,other method  need to walk the line of "gentle learning curve and great to customize".

**Comment by falkoschindler** [comment]
I could imagine a demo/example. Or maybe a builder-pattern-like method. But I wouldn't like to change the default behavior or add too many init parameters.
